pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -215,7 +215,7 @@ func (c *Cluster) makeMgrDaemonContainer(mgrConfig *mgrConfig) v1.Container {
 			"ceph-mgr",
 		},
 		Args: append(
-			controller.DaemonFlags(c.clusterInfo, mgrConfig.DaemonID),
+			controller.DaemonFlags(c.clusterInfo, mgrConfig.DaemonID, c.spec.Network.IPFamily),
```

Comment:
Please pass the whole spec so that we don't need to modify the signature of `DaemonFlags()` again in the future.

Reply:
passing the whole clusterSpec as the second argument